### PR TITLE
Limit and triggers gas costs

### PR DESCRIPTION
**Pull request by JameStark**
### Description of the Changes

Add more detail on costs for reads, writes and message sending to L1

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-901/documentation/

### Check List

- [ ] Changes have been done against dev branch, and PR does not conflict
- [ ] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/901)

### CODE REVIEW

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 1]
```diff
@@ -21,10 +21,19 @@ mechanism involves an Ethereum transaction for each Starknet block.
 
 The gas limit for Starknet blocks is therefore inherited from the gas limit for Ethereum blocks.
 |3M |3M
-|Max transaction size (Cairo steps)|The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.
 
+|Max transaction size (Cairo steps)|The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.
 This limit is important for ensuring the efficient execution of transactions and preventing potential congestion.
 | 3M |3M
+
+|Max Cairo steps for `validate`| The maximum number of computational steps, measured in Cairo steps for a `validate` function.  |1M |1M
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:29]:
```suggestion
|Max Cairo steps for `validate`| The maximum number of computational steps, measured in Cairo steps, for a `validate` function.  |1M |1M
```

**Review by stoobie — CHANGES_REQUESTED** [review]

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (JameStark) [reply components/Starknet/modules/tools/pages/limits_and_triggers.adoc:29]: Done

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 2]
```diff
@@ -16,7 +16,7 @@ These are subject to revisions and change on a regular basis
 |Block time | The average time it takes for a new block to be created on the Starknet blockchain.|3 minutes |3 minutes
 |Block limit (Cairo steps)|Starknet enforces a limit on the number of steps that can be completed
 within each block to ensure block production times remain consistent and predictable.  | 8M | 8M
-|Block limit (gas)| Certain Starknet operations consume Ethereum gas, e.g L1/L2 messages. The current L1 state update
+|Block limit (gas)| Certain Starknet operations, such sending messages between L1 and L2, consume Ethereum gas. The current L1 state update
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:19]:
```suggestion
|Block limit (gas)| Certain Starknet operations, such as sending messages between L1 and L2, consume Ethereum gas. The current L1 state update
```

**Review by stoobie — CHANGES_REQUESTED** [review]
Two suggestions awaiting response.

**Pull request merged**